torch_xla/csrc/tensor.h

```diff
@@ -314,15 +314,17 @@ class XLATensor {
                                     std::vector<xla::int64> stride,
                                     std::vector<xla::int64> padding);
 
-  static XLATensor conv_transpose2d(const XLATensor& input,
-                                    const XLATensor& weight,
-                                    std::vector<xla::int64> stride,
-                                    std::vector<xla::int64> padding);
-
-  static std::tuple<XLATensor, XLATensor, XLATensor> conv_transpose2d_backward(
-      const XLATensor& out_backprop, const XLATensor& input,
-      const XLATensor& weight, std::vector<xla::int64> stride,
-      std::vector<xla::int64> padding);
+  static XLATensor slow_conv_transpose2d(const XLATensor& input,
```

Comment:
We keep the overrides alphabetically ordered within this file. Can you drop them in the proper place?